sesman.el

```diff
@@ -624,8 +624,10 @@ types to consider. Defaults to the list returned from `sesman-context-types'."
 (defun sesman-current-session (system &optional cxt-types)
   "Get the most relevant current session for the SYSTEM.
 CXT-TYPES is a list of context types to consider."
-  (or (car (sesman--linked-sessions system 'sort cxt-types))
-      (car (sesman--friendly-sessions system 'sort))))
+  (if sesman-use-friendly-sessions
```

Comment:
This would seem DRYer

```
(defun sesman-current-session (system &optional cxt-types)
  "Get the most relevant current session for the SYSTEM.
CXT-TYPES is a list of context types to consider."
  (or (car (sesman--linked-sessions system 'sort cxt-types))
      (when sesman-use-friendly-sessions
        (car (sesman--friendly-sessions system 'sort)))))
```

Reply:
agreed. changed